linux/ibus-keyman/meson.build

```diff
@@ -0,0 +1,44 @@
+project('ibus-keyman', 'c', 'cpp',
+        version: run_command('cat', '../../VERSION.md', check: true).stdout().strip(),
+        license: 'GPL-2+',
+        # default_options : ['buildtype=release',
+        #                    'cpp_std=c++14',
+        #                    'b_vscrt=static_from_buildtype',
+        #                    'warning_level=2'],
+        meson_version: '>=0.53.0')
```

Comment:
linux/control gives >= 0.45. I would love to see 0.53+ or even higher?

Reply:
Yep, oversight. But it was changed in #7882 which has been merged in the meantime.